src/Normalizer/EntityReferenceItemNormalizer.php

```diff
@@ -109,6 +125,10 @@ public function normalize($field_item, $format = NULL, array $context = []) {
         // If there's no context, we need full predicates, not shortened ones.
         if (!$context['needs_jsonldcontext']) {
           $field_name = $this->escapePrefix($field_name, $context['namespaces']);
+          foreach ($values_clean as $key => $val) {
+            // Expand values in the array, ie. @type values xsd:string.
+            $values_clean[$key] = $this->escapePrefix($val, $context['namespaces']);
+          }
         }
         $normalized_prop[$field_name] = [$values_clean];
       }
```

Comment:
This is super kludgey, but adding
```php
    foreach($normalized_prop as $k => $v) {
      foreach($v as $k2 => $v2) {
        unset($normalized_prop[$k][$k2]['@type']);
      }
    }
```
at line 137 does the trick.

I'm sure there's better ways, but if all else fails, it'll do.